src/renderer/features/operation/sign/lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export enum ReconnectStep {
```

Comment:
Pure enums pollute bundle with worthless code
```suggestion
export const enum ReconnectStep {
```